tests/upgrade/raycluster_sdk_upgrade_test.py

```diff
@@ -0,0 +1,154 @@
+import requests
+from time import sleep
+
+from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication
+from codeflare_sdk.job import RayJobClient
+
+from tests.e2e.support import *
+from codeflare_sdk.cluster.cluster import get_cluster
+
+from codeflare_sdk.utils.kube_api_helpers import _kube_api_error_handling
+
+namespace = "test-ns-rayupgrade"
+
+
+# Creates a Ray cluster
+class TestMNISTRayClusterUp:
+    def setup_method(self):
+        initialize_kubernetes_client(self)
+        create_namespace_with_name(self, namespace)
+        try:
+            create_kueue_resources(self)
+        except Exception as e:
+            delete_namespace(self)
+            return _kube_api_error_handling(e)
+
+    def test_mnist_ray_cluster_sdk_auth(self):
+        self.run_mnist_raycluster_sdk_oauth()
+
+    def run_mnist_raycluster_sdk_oauth(self):
+        ray_image = get_ray_image()
+
+        auth = TokenAuthentication(
+            token=run_oc_command(["whoami", "--show-token=true"]),
+            server=run_oc_command(["whoami", "--show-server=true"]),
+            skip_tls=True,
+        )
+        auth.login()
+
+        cluster = Cluster(
+            ClusterConfiguration(
+                name="mnist",
+                namespace=self.namespace,
+                num_workers=1,
+                head_cpus="1",
+                head_memory=2,
+                min_cpus="1",
+                max_cpus=1,
+                min_memory=1,
+                max_memory=2,
+                num_gpus=0,
+                instascale=False,
```

Comment:
Just a small change, we can remove instascale now

Reply:
Yes, this update has been made.
Thank you @ChughShilpa for reviewing !